Tests/web3swiftTests/localTests/web3swiftDecodeSolidityErrorType.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  web3swiftDecodeSolidityErrorType.swift
```

Comment:
Please add this test to localTests.xctestplan

Reply:
Looks like it's already in it. Am I missing something?
<img width="668" alt="Screenshot 2022-03-28 at 14 35 44" src="https://user-images.githubusercontent.com/36865532/160389435-d950cac3-75ae-44f8-933d-f40080dac68e.png">

Reply:
Removed it from RemoteTests.xctestplan.
<img width="517" alt="Screenshot 2022-03-28 at 14 36 47" src="https://user-images.githubusercontent.com/36865532/160389592-48d185c4-41f3-482a-9f87-ecf8bfd5b5ac.png">